react/DocProp.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react'
+
+import PropsTable from './components/PropsTable'
+
+import useComponentSchema from './hooks/useComponentSchema'
+import { Spinner, Alert } from 'vtex.styleguide'
+
+const DocProp: FC<DocPropProps> = ({ blockInterface }) => {
+  const appId = window.__RUNTIME__.route.params.app
```

Comment:
use `vtex.render-runtime`'s  `useRuntime` hook instead of using `window`

Example [here](https://github.com/vtex-apps/recommendation-shelf/blob/ec5edd6ee7288f738fd83831eb4d175edcd84e6d/react/utils/useUserNavigation.js#L10)